rgd/utility.py

```diff
@@ -240,3 +241,11 @@ def patch_internal_presign(f: FieldFile):
             f.storage.base_url = original_base_url
         return
     yield
+
+
+def uuid_prefix_filename(instance: Any, filename: str):
+    """Use a variable in settings to add a prefix to the path and keep the random uuid."""
+    prefix = getattr(settings, 'S3_FILE_FIELD_PREFIX', None)
```

Comment:
I think you should name this setting differently, since it's part of RGD, not S3FF.

Reply:
Fair point, how about `RGD_FILE_FIELD_PREFIX`?